Documentation/General/finalReport/InteractionDesign.tex

```diff
@@ -1,2 +1,106 @@
-\chapter{Human-Computer Interaction }
+\chapter{Human-Computer Interaction}
+
+\section{Introduction}
+This section describes the interaction between the user and the computer and how we measured this.
+The user is very important for us, the user is the one that should use the application.
+Our system is build for interaction between users and is improving by user interaction.
+The interaction between users is the chatroom that our whole music application is build around.
+The music itself is based on valence and arousal witch is classified by the user.
+So, the more users our system has the better it will work.
+
+
+%Beschrijving van de gebruiker en gebruiksituatie
+
+%• Gebruikersverhalen: duidelijk maken wat mensen willen en doen (0)
+%• Conceptuele scenario's: genereren van ideeën en vaststellen van requirements (0)
+%• Concrete scenario’s: Gebruik maken van prototypes en voor evaluatie (1)
+%• Use cases: voor documentatie en implementatie van interactie (3)
+
+\section{The user and their situation}
+
+\subsection{Use case scenarios}
+Here are a few use case scenarios, we have made more.
```

Comment:
Obviously we thought of more use cases, but we have not documented them. Furthermore, I would not say: "Here are a few use case scenarios."
I would suggest to remove this sentence, and say something like:
> In this chapter we will elaborate the use cases we have made for our product.